src/components/Human/Human.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person;
+  personMother?: Person;
+  personFather?: Person;
+  personSlug?: string;
```

Comment:
```suggestion
  selectedSlug?: string;
```
- You can store `mother` and `father` directly in the `person` object
- `personSlug` is a bit confusing as `person` itself has a slug